src/Routes.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {Route} from 'react-router-dom';
+
+const components = [{
+  urlPath: 'button',
+  filePath: './ButtonCatalog',
+}, {
+  urlPath: 'card',
+  filePath: './CardCatalog',
+}, {
+  urlPath: 'checkbox',
+  filePath: './CheckboxCatalog',
+}, {
+  urlPath: 'chips',
+  filePath: './ChipsCatalog',
+}, {
+  urlPath: 'dialog',
+  filePath: './DialogCatalog',
+}, {
+  urlPath: 'drawer',
+  filePath: './DrawerCatalog',
+}, {
+  urlPath: 'elevation',
+  filePath: './ElevationCatalog',
+}, {
+  urlPath: 'fab',
+  filePath: './FabCatalog',
+}, {
+  urlPath: 'icon-button',
+  filePath: './IconButtonCatalog',
+}, {
+  urlPath: 'image-list',
+  filePath: './ImageListCatalog',
+}, {
+  urlPath: 'layout-grid',
+  filePath: './LayoutGridCatalog',
+}, {
+  urlPath: 'list',
+  filePath: './ListCatalog',
+}, {
+  urlPath: 'linear-progress-indicator',
+  filePath: './LinearProgressIndicatorCatalog',
+}, {
+  urlPath: 'menu',
+  filePath: './MenuCatalog',
+}, {
+  urlPath: 'radio',
+  filePath: './RadioButtonCatalog',
+}, {
+  urlPath: 'ripple',
+  filePath: './RippleCatalog',
+}, {
+  urlPath: 'select',
+  filePath: './SelectCatalog',
+}, {
+  urlPath: 'shape',
+  filePath: './ShapeCatalog',
+}, {
+  urlPath: 'slider',
+  filePath: './SliderCatalog',
+}, {
+  urlPath: 'snackbar',
+  filePath: './SnackbarCatalog',
+}, {
+  urlPath: 'switch',
+  filePath: './SwitchCatalog',
+}, {
+  urlPath: 'tabs',
+  filePath: './TabsCatalog',
+}, {
+  urlPath: 'text-field',
+  filePath: './TextFieldCatalog',
+}, {
+  urlPath: 'theme',
+  filePath: './ThemeCatalog',
+}, {
+  urlPath: 'top-app-bar',
+  filePath: './TopAppBarCatalog',
+}, {
+  urlPath: 'typography',
+  filePath: './TypographyCatalog',
+}];
+
+const Routes = () => {
+  return (
+    components.map((component) => {
+      const {filePath, urlPath} = component;
+      const Component = require(`${filePath}`).default;
```

Comment:
@kfranqueiro since this is dynamic we cannot use `import`. `import` statements also must go at the top of the file.

We could do 
```
import ButtonCataglog from './ButtonCatalog';
...

components = [{
  urlPath: 'button',
  component: ButtonCatalog,
...
}];
```

But this doesn't really seem to save us anything.

Reply:
Yes, your "we could do" is what I was referring to, and it saves us from doing a hack of non-ES2015 imports inside an otherwise ES2015 module.